### PR TITLE
Fix pom file versions and comments

### PR DESCRIPTION
**Pull request by sgilda**
Please review carefully! 
I will create bugs for upstream quickstarts affected by these changes.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -240,6 +268,7 @@
             <modules>
                 <module>forge-from-scratch</module>
                 <module>h2-console</module>
+                <module>kitchensink-cordoba</module>
```
Comment (rafabene) [inline pom.xml]: it is cordova instead of cordoba

File: pom.xml [hunk 2]
```diff
@@ -193,6 +220,7 @@
             </activation>
             <modules>
                 <module>cluster-ha-singleton</module>
+                <module>helloworld-jdg</module>
```
Comment (rafabene) [inline pom.xml]: I think helloworld-jdg could be on default profile. I don't think it is complex-dependencies

File: pom.xml [hunk 3]
```diff
@@ -240,6 +268,7 @@
             <modules>
                 <module>forge-from-scratch</module>
                 <module>h2-console</module>
+                <module>kitchensink-cordoba</module>
                 <module>jts-distributed-crash-rec</module>
```
Comment (rafabene) [inline pom.xml]: It' is cordova instead of cordoba.

**Comment by rafabene** [comment]
Finished review. Had come comments.

**Comment by sgilda** [comment]
@rafabene : Great catch on both of those! Turns out the helloworld-jdg was failing for the same reasons carmart and carmart-tx were failing. I also removed <module>cmt</module> from the default profile. It doesn't belong there. This is ready for review again. :-)

**Comment by rafabene** [comment]
It seems fine!

**Pull request merged**